docs/detections/add-exceptions.asciidoc

```diff
@@ -43,12 +43,17 @@ image::images/rule-exception-tab.png[Detail of rule exceptions tab]
 .. Go to *Alerts*.
 .. Scroll down to the Alerts table, go to the alert you want to create an exception for, click the *More Actions* menu (*...*), then select *Add rule exception*.
 
+* To add an exception from the alert details flyout:
+.. Go to *Alerts*.
+.. Click the *View details* button from the Alerts table. 
+.. In the alert details flyout, click *Take action -> Add rule exception*. 
+
 * To add an exception from the Shared Exception Lists page:
 .. Go to *Manage* -> *Shared Exception Lists*.
 .. Click *Create shared exception list* -> *Create exception item*. 
 --
 
-. In the *Add rule exception* flyout, name the exception and add conditions that define when the exception prevents alerts. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
+. In the *Add rule exception* flyout, name the exception and add conditions that define the exception. When the exception's query conditions are met (the query evaluates to `true`), rules do not generate alerts even when other rule criteria are met.
```

Comment:
Nit: @nastasha-solomon I'm curious about the reason for this edit. The original is more detailed in describing what exceptions and conditions do (prevent alerts), but omitting that just leaves "conditions that define the exception" which is pretty generic.

Reply:
Good dialogue! I see both sides, but I think the second sentence that says `rules do not generate alerts even when other rule criteria are met` imply that that's what an exception does (prevents alerts), so I'm okay with the concise edit here.